DEVELOPMENT.md

```diff
@@ -62,7 +62,34 @@ $ pytest -v
 Rust unit tests can be run with `cargo`:
 
 ```shell
-$ cargo test --all
+$ cargo test --workspace --exclude rustpython_wasm
+```
+
+Python unit tests can be run by compiling RustPython and running the test module:
+
+```shell
+$ cargo run --release -- -m test
+```
+
+There are a few test options that are specially useful:
```

Comment:
nitpick (grammar):
```suggestion
There are a few test options that are especially useful:
```

Reply:
Thanks! I actually didn't know the difference as a non-native speaker